software/README.md

```diff
@@ -49,26 +49,34 @@ If you really want to avoid 'noise' which would present as a flickering value de
 The ADCs used to read the knob position are only 12 bit, which means that any read_position value above 4096 (2^12) will not actually be any finer resolution, but will instead just go up in steps. For example using 8192 would only return values which go up in steps of 2.
 
 
-## Digital Input and Buttons
+## Digital Input
 
-It may seem a little strange to see the digital input and buttons grouped together, but if you think about it, all the button is doing is create an internal digital input when you press it. For this reason, the method for handling the event that should happen when either a button is pressed or a digital input detected is the same.
-
-A class is used named 'DigitalInput' in the europi.py library, which handles both the debounce (only used for buttons) and the handler (what happens when a press/input is detected).
+The Digital Input jack can detect a HIGH signal when recieving voltage > 0.8v and will be LOW when below.
 
 | Method        | Usage       | Parameter(s)       |
 | ------------- | ----------- | ----------- |
 |value|Reads the current value of the input, HIGH (1) or LOW (0).|
 |handler|Assign a new function to be used as the handler|function
 |reset_handler|Detach the handler method from the Pin IRQ|
+|last_trigger|Return the time in milliseconds of the last trigger.|
 
 It should be noted that the value will be 0 when the input is 'high', and 1 when 'low'. This is simply a hardware technicality, but will not affect the use of the handler, only the .value() mathod.
```

Comment:
Accidentally highlighted the wrong part, I'm referring to the sentence on line 63 about the inverted digital input behaviour, but this has been obscured from the user now so it will display 'properly' aka 1 = High and 0 = Low

Reply:
Good catch, removed.